js/photoeditor.js

```diff
@@ -77,6 +77,18 @@ var photoEditor = {
         document.getElementById('pe-undoBttn').onclick = this.undo.bind(this);
         document.getElementById('pe-redoBttn').onclick = this.redo.bind(this);
 
+        //fix by team wp
```

Comment:
can you remove this line, please?
it is unnecessary.

Reply:
yes, i can.